pom.xml

```diff
@@ -125,6 +125,9 @@ under the License.
 		<powermock.version>1.6.5</powermock.version>
 		<hamcrest.version>1.3</hamcrest.version>
 		<japicmp.skip>false</japicmp.skip>
+		<!-- empty placeholder for the flip6 build profile -->
```

Comment:
nit: I don't understand this comment. Shouldn't it be something like 
```<!-- run all groups except flip6 by default -->``` ?

Reply:
Yes this better. I'll change it.